### ISSUE TITLE
Selectable Tile is not accessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6895]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description

The focus state and icons indicating a selection

Firefox 
![image](https://user-images.githubusercontent.com/40970507/93946072-ec52db80-fcfd-11ea-93ec-9d5cbc02d4de.png)

Edge
![image](https://user-images.githubusercontent.com/40970507/93946075-f07ef900-fcfd-11ea-815c-618185946aed.png)
